.gitignore

```diff
@@ -129,6 +130,7 @@ node_modules
 !.yarn/versions
 .pnp.*
 yarn-error.log
+yarn.lock
```

Comment:
Shouldn't be added to the .gitignore. Please use yarn as default package manager.